wagtail/project_template/home/models.py

```diff
@@ -1,4 +1,4 @@
-from django.db import models
+from django.db import models  # lgtm [py/unused-import]
```

Comment:
For the project template I think we should remove this so the template output passes basic linting, and we don’t risk littering other people’s codebases with ignore comments for LGTM.